### ISSUE TITLE
The performance on your paper cannot be repeated, may you share the training parameters you used

### ISSUE THREAD

**Issue by jchhuang** [tngh5004/Omnistitch#13]
When running the train.py and using the default parameters on your released dataset GV360, the obtained inference result (as attachment，very blur and distorted) is considerable different from that‘s of your paper，so may you share the finally used training parameters？
![3-interp-img-new2](https://github.com/user-attachments/assets/ae7888b4-9dc9-459a-a255-26c85f5d8983)

**Comment by tngh5004** [comment]
Hello, @jchhuang  
The weight (parameter) files published in the paper are already publicly available at the repository address below.  
[train-log-/Omnistitch/trained-models/model.pkl](https://github.com/tngh5004/Omnistitch/blob/main/train-log-/Omnistitch/trained-models/model.pkl)

**Comment by jchhuang** [comment]
Yes, there is indeed a model file here. But my question is I cannot repeat the training process to obtain a model with the same performance as yours. So I hope you can share the training parameters (e.g. init_lr、min_lr and steps), so that I can repeat the training process and then can make some optimizations on your work. Thanks a lot.
